src/MarkerClusterAsset.php

```diff
@@ -32,7 +32,7 @@ class MarkerClusterAsset extends AssetBundle
 
     public function init()
     {
-        $this->sourcePath = __DIR__ . '/assets';
-        $this->js = YII_DEBUG ? ['js/leaflet.markercluster-src.js'] : ['js/leaflet.markercluster.js'];
+        $this->sourcePath = \Yii::getAlias('@bower').'/leaflet.markercluster/dist';
+        $this->js = YII_DEBUG ? ['leaflet.markercluster-src.js'] : ['leaflet.markercluster.js'];
```

Comment:
I don't mind to make the change, but I didn't get what is wrong with the current PR.